pkgs/development/mobile/genymotion/default.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
   name = "genymotion-${version}";
   version = "2.8.0";
   src = requireFile {
-    url = https://www.genymotion.com/download/;
+    url = https://dl.genymotion.com/releases/genymotion-2.8.0/genymotion-2.8.0-linux_x64.bin;
```

Comment:
Since you're at it, the url type will probably go away soon and should be a string.

So my suggestion is:
```nix
     url = "https://dl.genymotion.com/releases/genymotion-${version}/genymotion-${version}-linux_x64.bin";
```

That way we probably don't have to change the URL whenever there's a new version, just bump the version.